### PR TITLE
Created docs

### PR DESCRIPTION
**Pull request by vojtechmares**
Hi,
as I said yesterday at PoSobota event, I have written the docs for this repo.

I hope it is ok.

Edit: closed & reopened because I forgot to check `.gitattributes`. But it is ok.

### CODE REVIEW

File: .docs/README.md [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+
+To use these commands we recommend to use an **[entrypoint](https://github.com/contributte/console/blob/master/.docs/README.md#entrypoint)** or you can use your `index.php`.
+
+With [entrypoint](https://github.com/contributte/console/blob/master/.docs/README.md#entrypoint):
+```bash
+./bin/console <command>
+```
+without [entrypoint](https://github.com/contributte/console/blob/master/.docs/README.md#entrypoint):
+```bash
+php public/index.php <command>
```
Comment (f3l1x) [inline .docs/README.md]: What does it mean without entrypoint? :-)

Comment (vojtechmares) [reply .docs/README.md]: it sounds weird now. I ment it without `bin/console` over the `index.php`, but from the definition it is entrypoint either, going to fix it with different words

Comment (f3l1x) [reply .docs/README.md]: Index.php is not supported. Its fron the old times in Kdyby/Console.

**Comment by vojtechmares** [comment]
@f3l1x does anything need some fixes? If not, can you merge it please?

**Comment by f3l1x** [comment]
Thank you.

**Pull request merged**